src/contracts/proxies/actions/CommonActions.sol

```diff
@@ -3,6 +3,7 @@ pragma solidity 0.8.19;
 
 import {ISAFEEngine} from '@interfaces/ISAFEEngine.sol';
 import {IERC20Metadata} from '@openzeppelin/token/ERC20/extensions/IERC20Metadata.sol';
+import {IERC20MetadataUpgradeable} from '@openzeppelin-upgradeable/token/ERC20/extensions/IERC20MetadataUpgradeable.sol';
```

Comment:
Why is it that we have to import the new interface everywhere? Can we not just use the existing interface, since these contracts don't rely on calling the initialize function?

Reply:
Yes, this is a very annoying issue, but basically compiler does not view upgradeable and standard contracts the same because of the inherited contracts, which are also divided as upgradeable and standard.